evennia/utils/ravensgleaning.py

```diff
@@ -0,0 +1,441 @@
+"""
```

Comment:
The name of this module means nothing for Evennia users. We should reference the original name and accredit properly in the docstring, but the module should be renamed to something more usable, such as `ansi2html.py` (so as to not collide with the existing `text2html.py`. 

Also, as far as I understand, this is more complete/produces better results than the old `text2html.py` implementation? Are there unit tests to prove this?

Reply:
It is as close to perfect as you can get. Evennia's own implementation cannot handle arbitrary; it can only handle the output produced by ANSIString, which it does by pre-defining classes. RavensGleaning defines spans with inline styles specific to the ANSI state. So it's more accurate since it doesn't need predefined CSS classes.